include/material/acoustic_material.hpp

```diff
@@ -48,8 +73,8 @@ class acoustic_material : public material {
    * @brief Acoustic material properties
    *
    */
```

Comment:
@williameaton can you edit this to look similar to elastic class? Only one variable per line for readability